### ISSUE TITLE
The format of the email address output in Japanese is not correct.

### ISSUE THREAD

**Issue by shu-pf** [faker-js/faker#1437]
### Pre-Checks

- [X] Follow our [Code of Conduct](https://github.com/faker-js/faker/blob/next/CODE_OF_CONDUCT.md).
- [X] Read the [Contributing Guidelines](https://github.com/faker-js/faker/blob/next/CONTRIBUTING.md).
- [X] Read the [docs](https://fakerjs.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/faker-js/faker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Faker issue and not related to a combination with another package.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/faker-js/faker/discussions) or join our [Discord Chat Server](https://chat.fakerjs.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.
- [X] I am willing to provide a PR.

## Clear and concise description of the problem

Japanese people do not use multibyte characters in their e-mail addresses because using multibyte characters in some e-mail addresses can cause erroneous transmissions.
Therefore, we generally convert the name to romaji first and use that for the e-mail address.

## Suggested solution

Current Behavior

```
import { faker } from '@faker-js/faker/locale/ja';

const email = faker.internet.email();
console.log(email); // 樹_林@hotmail.com
```

Expected behavior

```
import { faker } from '@faker-js/faker/locale/ja';

const email = faker.internet.email();
console.log(email); // itsuki_hayashi@hotmail.com
```

### Additional Context

It may also be possible to solve this problem by creating a function that converts Japanese names to romaji.

### Environment Info

```shell
System:
    OS: macOS 11.6
    CPU: (8) arm64 Apple M1
    Memory: 133.45 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.13.2 - ~/.volta/tools/image/node/16.13.2/bin/node
    Yarn: 1.22.19 - ~/.volta/tools/image/yarn/1.22.19/bin/yarn
    npm: 8.1.2 - ~/.volta/tools/image/node/16.13.2/bin/npm
  Browsers:
    Chrome: 106.0.5249.103
    Firefox: 96.0.1
    Safari: 15.3
  npmPackages:
    @faker-js/faker: ^7.5.0 => 7.5.0
```

### Which module system do you use?

- [ ] CJS
- [X] ESM

### Used Package Manager

yarn

**Comment by Shinigami92** [comment]
> It may also be possible to solve this problem by creating a function that converts Japanese names to romaji.

We do not provide utility functions that are not based on the underlying seed, but you might have luck with e.g. https://www.npmjs.com/package/hepburn and pass the values to the `internet.email` function.

Please use that for a workaround for now as we are now in a maintenance state with faker to move forward to v8.
So unless something it really critical we wont have the time and resources to fix and release this proposal.

**Comment by shu-pf** [comment]
Thanks for the reply.

I checked Hepburn.
This library can convert Hiragana and Katakana, but not Kanji.
The names provided by Faker are in Kanji. It seems difficult to solve the problem here.

**Comment by Shinigami92** [comment]
> Thanks for the reply.
> 
> I checked Hepburn. This library can convert Hiragana and Katakana, but not Kanji. The names provided by Faker are in Kanji. It seems difficult to solve the problem here.

@shu-pf The linked package was just a quick search with a search engine. I assume you are smart enough to find a suitable npm package for your need very quickly 😉 
The process would still be the same: transform and use that as input

**Comment by shu-pf** [comment]
@Shinigami92 
I have not checked all packages, but I don't think there is a package that converts Kanji to Romaji.
Conversion between kanji and romaji cannot be done mechanically, becose there are multiple patterns and some conversions that are not listed in the dictionary.
Hiragana has a one-to-one relationship with romaji, so it is assumed that a package exists.

So it would be necessary to manually set up the relationship between the Kanji name and the Romaji name to fix this feature.

**Comment by Shinigami92** [comment]
> Conversion between kanji and romaji cannot be done mechanically, because there are multiple patterns and some conversions that are not listed in the dictionary.

So how in the world should Faker then provide a functionality for that 👀 
I don't see the responsibility on our side for doing that 🤔 
Currently emails will be generated just by first and last name.
Do you have any proposal how to handle that for japanese instead of converting kanji and romaji?

**Comment by ST-DDT** [comment]
I think we could base32(?) encode the incompatible characters. That wont yield nice email addresses, but usable and reproducible ones.

**Comment by shu-pf** [comment]
@Shinigami92 

I think the only way is to make a table of correspondence between kanji and romaji and use it.

I don't have a deep understanding of the current code, but if I were to write the code. 
First prepare two tables.

last_name.ts

```js
export default [
  '佐藤',
  '鈴木',
  ...
];
```

last_name_romaji.ts

```js
export default [
  'Satou',
  'Suzuki',
  ...
];
```

And for names, use the 'last_name.ts'.
For e-mail, use the 'last_name_romaji.ts'.

It takes a lot of effort to have this done by someone other than a native Japanese speaker, so of course I am willing to help.

@ST-DDT 
Thank you for sharing your solution! I hadn't thought of that.
But I think base32 is too long for email. I use nanoid package for this.

It seems to be a realistic solution at this time.

**Comment by xDivisionByZerox** [comment]
This has likely been fixed by #1554 and is part of https://github.com/faker-js/faker/releases/tag/v8.0.0-alpha.0.
If this is not the case please reopen this issue.
